app/qml/PositionMarker.qml

```diff
@@ -32,14 +88,16 @@ Item {
         id: direction
         source: "gps_direction.svg"
         fillMode: Image.PreserveAspectFit
-        rotation: positionKit.direction
+        rotation: positionMarker.direction
         transformOrigin: Item.Bottom
         width: positionMarker.size * 2
         height: width
         smooth: true
-        visible: positionKit.hasPosition && positionKit.direction >= 0
+        visible: positionKit.hasPosition
```

Comment:
why remove `&& positionKit.direction >= 0` ... if we have no speed and no compass reading... we would be showing direction=-1 ? Probably better not to make it invisible